### PR TITLE
[cluster-test] Integrate with config builder

### PR DESCRIPTION
**Pull request by andll**
This diff prepares cluster test so it can work with workplace set up with config-builder
Config-builder <-> cluster-test api is not ideal probably, but it fixes an issue. We can re-iterate on better API later

**Please use `delegate+`, still need to verify those changes**

### CODE REVIEW

File: terraform/variables.tf [hunk 1]
```diff
@@ -39,6 +39,7 @@ variable "image_tag" {
   default     = "latest_dynamic"
 }
 
+# This var is used by cluster test in cluster.rs, please update in both places if this value changes
```
Comment (ankushagarwal) [inline terraform/variables.tf]: I'm willing to bet 5 Libra USD that this will break cluster-test at least once :)

**Comment by andll** [comment]
@ankushagarwal I think long term cluster-test will need to work with ECS directly, including setting `--seed` command line. This way it will be less likely to break.
For now I just left comment, if people won't just ignore it, this should be fine for time being

**Pull request closed without merging**